.github/ISSUE_TEMPLATE/1.bug.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Bug report
+about: Create a bug report for Chirpstack
```

Comment:
Maybe change _Chirpstack_ to _ChirpStack Gateway Bridge_ to make sure that people understand that this is specific to this component?